src/components/reusables/SharedStylingV2.js

```diff
@@ -192,6 +192,7 @@ export const ImageV2 = styled.img`
   overflow: ${(props) => props.overflow || 'initial'};
   cursor: ${(props) => props.cursor || 'default'};
   filter: ${(props) => props.filter || 'none'};
+  alt: ${(props) => props.alt || ''};
```

Comment:
In case of empty alt prop, put some default string(maybe, 'Image') instead of empty string.